graph_altimeter/scan/__init__.py

```diff
@@ -34,12 +35,17 @@
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 
+arn_aws_account_re = re.compile('^arn:aws:iam::([0-9]{12}):root$')
```

Comment:
s/arn_aws_account_re/aws_account_arn_re/

So it is consistent with `aws_account_id_re`.